test/LibraryManager.IntegrationTest/AddCientSideLibrariesFromUITests.cs

```diff
@@ -0,0 +1,120 @@
+﻿using System.IO;
+using Microsoft.Test.Apex.VisualStudio.Shell.ToolWindows;
+using Microsoft.Test.Apex.VisualStudio.Solution;
+using Microsoft.VisualStudio.TestTools.UnitTesting;
+using Microsoft.Web.LibraryManager.IntegrationTest.Services;
+
+namespace Microsoft.Web.LibraryManager.IntegrationTest
+{
+    [TestClass]
+    public class AddCientSideLibrariesFromUITests : VisualStudioLibmanHostTest
```

Comment:
>AddCientSideLibrariesFromUITests [](start = 17, length = 32)

nit: typo

Reply:
Updated in iteration 7

---
In reply to: [206365193](https://github.com/aspnet/LibraryManager/pull/332#discussion_r206365193) [](ancestors = 206365193)